_api-reference/security/authentication/change-password.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: default
+grand_title: Change Password API
+grand_parent: Security APIs
+parent: Authentication APIs
+nav_order: 30
+---
+
+# Change Password API
+**Introduced 1.0**
+{: .label .label-purple }
+
+The Change Password API allows users to update their own passwords. This API provides a secure way for users to manage their credentials without requiring administrator intervention. Users must provide their current password for verification before the password change is allowed.
```

Comment:
nit: omit `without requiring administrator intervention`, being able to change one's own pw without administrator intervention is expected behavior of any system. Usually administrators only need to intervene after a user is locked out (not opensearch specific)